proposed_solutions/core.clj

```diff
@@ -0,0 +1,77 @@
+;from core.clj by Larissa Gaulia
+
+(ns cipher.core)
+
+(defn abs
+  [x y]
+  (if (< x y) (- y x) (- x y)))
```

Comment:
Actually this is not exactly an `abs` function, not that the function is wrong but is just a name thing